inst/private/python_ipc_pytave.m

```diff
@@ -70,7 +70,6 @@
 
   s = strjoin(cmd, newl);
   pyexec(s)
-  
-  info.raw = pyeval('str(_outs)');
```

Comment:
> Actually I still don't know how this is used.

Its just a hack that @latot came up with: reports the line number an 
exception occurred within the `cmd` block of `python_cmd`.  But 
different IPC mechanisms added various amounts of stuff to `cmd`. 
`prelines` keeps track of how much stuff.  Its probably zero for PyTave ipc.

Rest of this look ok to merge?